src/tests/ftest/nvme/enospace.yaml

```diff
@@ -45,6 +45,7 @@ ior:
   iorflags:
     flags: "-w -F -k -G 1"
     read_flags: "-r -R -F -k -G 1"
+  repetitions: 100
```

Comment:
This affects multiple ior commands - more than the two we want to change:
```suggestion
```

Reply:
ok